pybossa/view/projects.py

```diff
@@ -1698,6 +1699,110 @@ def bulk_priority_update(short_name):
     except Exception as e:
         return ErrorStatus().format_exception(e, 'priorityupdate', 'POST')
 
+@blueprint.route('/<short_name>/tasks/assign-workersupdate', methods=['POST'])
+def bulk_update_assign_worker(short_name):
+    import copy
+    from sqlalchemy.orm.attributes import flag_modified
+
+
+    response = {}
+    project, owner, ps = project_by_shortname(short_name)
+    data = json.loads(request.data)
+
+    if data.get("add") is None and data.get("remove") is None:
```

Comment:
This if block seems related to `GET` request, hence can be replaced w/
```python
if request.method == 'GET':
```
This will also need to updated allowed methods at line 1702 as
```python
... methods=['GET', 'POST']
```

Reply:
Paxton, the condition 
if data.get("add") is None and data.get("remove") is None: 
can be replaced with
if request.method == 'GET':
Also it seems there's scope of some refactoring consolidating common code. We can discuss this over call